ckanext/report/controllers/__init__.py

```diff
@@ -38,6 +38,9 @@ def report_view(report_name, organization=None, refresh=False):
         t.abort(401)
     except t.ObjectNotFound:
         t.abort(404)
+    except Exception as e:
+        log.error("Failed to get report: %s", e)
```

Comment:
so what error should it be, or are you just letting it return a 500 or other error?

Reply:
It's just some extra logging.